tests/isolation_provider/test_dummy.py

```diff
@@ -25,19 +25,20 @@ def start_doc_to_pixels_proc(self, document: Document) -> subprocess.Popen:
             stdin=subprocess.PIPE,
             stdout=subprocess.PIPE,
             stderr=subprocess.PIPE,
+            start_new_session=True,
         )
 
-    def terminate_doc_to_pixels_proc(
-        self, document: Document, p: subprocess.Popen
-    ) -> None:
-        p.terminate()
-
 
 @pytest.fixture
 def provider_wait() -> DummyWait:
```

Comment:
Reading "provider_wait" here didn't made it clear to me on first sight. Maybe we could name this `blocked_provider` :thinking: ?

Reply:
Good point. Actually, we will fully remove this in a PR I'll send soon, so let's keep it as is for now, to avoid merge conflicts.